### PR TITLE
Include external Python dependencies.

### PR DESCRIPTION
**Pull request by mtsgrd**
- Removes filter that excludes external sources.
- Fixes "package" import candidates.

Fixes issue https://github.com/bazelbuild/intellij/issues/228

### CODE REVIEW

**Comment by anishk25** [comment]
Ping on this for review. Would love to see this change released in the next version.

**Comment by dgarzon** [comment]
@brendandouglas @mtsgrd Hi, what's the status on this? Is it feature complete, and will it land soon?

**Comment by dgarzon** [comment]
@brendandouglas @mtsgrd if this is not ready, I would like to volunteer to complete the work. Not having external dependencies / IntelliSense for Python is a deal breaker for my team to complete our migration to Bazel 👎

**Comment by mtsgrd** [comment]
Since nobody from bazelbuild is answering, how about you guys compile and distribute a custom version to your developers?

File: python/src/com/google/idea/blaze/python/resolve/provider/AbstractPyImportResolverStrategy.java [hunk 1]
```diff
@@ -80,6 +80,16 @@ public final void addImportCandidates(
       PsiFile file = psi.getContainingFile();
       if (file != null) {
         quickFix.addImport(psi, file, candidate.removeLastComponent());
+        continue;
```
Comment (jin) [inline python/src/com/google/idea/blaze/python/resolve/provider/AbstractPyImportResolverStrategy.java]: (HT @brendandouglas)

Please use `PyUtils.turnDirIntoUtil(resolver.get(psiManager))` instead, so these entire diff is not needed.

**Comment by jin** [comment]
@mtsgrd if you have the time, please address the review comments and I'll import this. If not, I'll go ahead and apply a manual patch EOD tomorrow.

**Pull request closed without merging**